### DISCUSSION TITLE
Large Files - hpt-validator-cli Install

### DISCUSSION THREAD

**Discussion by mcrisanSP** [CMSgov/hospital-price-transparency#122] [Q&A] [answered]
I have some large .csv files that are crashing the online validator. The validator install expands beyond my depth of tech knowledge. I cannot figure out where to find the hpt-validator-cli install package. I reached out to my IT support and they cannot figure it out either. Can anyone point me in the right direction? I've validated my large files in pieces, but I'd like to run them through as one. Thank you

**Comment by CarlsonErin** [comment, marked as answer]
There are no formal file size limitations for the online validator. You may experience longer processing times, depending on, for example, the size and format of your machine-readable file and the available memory of the machine you are using. If you need to test a large file, we
recommend you use the command-line interface (CLI) validator instead of the online validator. Information on how to install the Validator-CLI are available at: https://github.com/CMSgov/hpt-validator-cli?tab=readme-ov-file.  Releases are available in the far right column.
